docs/modules/develop/pages/manual/operations/deployment.adoc

```diff
@@ -28,13 +33,23 @@ We'll need to set up an array of environment variables in order for the sever to
 * Rails:
  ** `RACK_ENV` and `RAILS_ENV` with value `production`;
  ** `RAILS_SERVE_STATIC_FILES` with value `true`;
- ** `SECRET_KEY_BASE`.
+ ** `SECRET_KEY_BASE` as in any Rails application;
 * Connect to Postgres:
  ** `DATABASE_URL` with the full URL of the database;
 * Redis:
  ** `REDIS_URL` with the full URL of the Redis instance;
 * ActiveRecord:
  ** The secrets of the cloud storage of your choice;
+ ** For instance, if using a compatible AWS provider you should define:
+   *** `ACTIVE_STORAGE_SERVICE=s3`
+   *** `AWS_ACCESS_KEY_ID=********`
+   *** `AWS_SECRET_ACCESS_KEY=********`
+   *** `AWS_REGION=eu-west-1`
+   *** `AWS_BUCKET_NAME=bucket-name`
+   *** If using a provider that is not Amazon also define `AWS_ENDPOINT` with the endpoint of the provider.
+ ** You can also easily use Azure or GCS providers (see the previous mentioned `storage.yml` file for the ENV vars), but in this case add the line that applies in your case to the file `bulletin_board/server/Gemfile`:
+   *** `gem "azure-storage-blob", require: false`
+   *** `gem "google-cloud-storage", "~> 1.11", require: false`
```

Comment:
Can we add it commented out in the Gemfile? Also, can you add the `azure-storage-blob` version too?

Reply:
sure, as a note, there's no version in the rails documentation when recomend adding this line

Reply:
Ok, if the docs don't have it then I can live with that